init.vim

```diff
@@ -193,3 +194,6 @@ let g:ale_fixers = {
 
 " ALE quickfix with Prettier
 nmap <leader>e <Plug>(ale_fix)
+
+" Clear search highlight
+nnoremap <silent> <leader>l :nohlsearch<CR>
```

Comment:
I already have a mapping for this `nnoremap <silent> <leader><cr> :let @/ = ""<cr>` which is a more correct way to clear search highlight.